src/Search/Processors/SearchUpdateProcessor.php

```diff
@@ -154,6 +155,9 @@ protected function getSource()
      */
     public function process()
     {
+        if (!DB::is_active()) {
```

Comment:
What prompted this? It doesn't seem related to the rest of the PR?

Reply:
PR that went into 3.9 though wasn't merged up - https://github.com/silverstripe/silverstripe-fulltextsearch/pull/318